tests/manifold/spherical_manifold_01.output

```diff
@@ -1,381 +1,80 @@
-81 5 0 0 8 13 61 121 64 62 122 149 123 
-82 5 0 0 8 61 1 30 121 122 29 91 149 
-83 5 0 0 8 62 122 149 123 3 35 98 37 
-84 5 0 0 8 122 29 91 149 35 9 50 98 
-85 5 0 0 8 64 121 87 27 123 149 137 88 
-86 5 0 0 8 121 30 15 87 149 91 67 137 
-87 5 0 0 8 123 149 137 88 37 98 71 17 
-88 5 0 0 8 149 91 67 137 98 50 23 71 
-89 5 0 0 8 2 32 97 34 33 95 150 96 
-90 5 0 0 8 32 11 58 97 95 55 119 150 
-91 5 0 0 8 33 95 150 96 13 61 121 64 
-92 5 0 0 8 95 55 119 150 61 1 30 121 
-93 5 0 0 8 34 97 69 16 96 150 129 70 
-94 5 0 0 8 97 58 25 69 150 119 83 129 
-95 5 0 0 8 96 150 129 70 64 121 87 27 
-96 5 0 0 8 150 119 83 129 121 30 15 87 
-$ENDELM
+DEAL::=================================
+DEAL::0.195090 0.980785
+DEAL::0.382683 0.923880
+DEAL::0.555570 0.831470
+DEAL::0.707107 0.707107
+DEAL::0.831470 0.555570
+DEAL::0.923880 0.382683
+DEAL::0.980785 0.195090
+DEAL::=================================
+DEAL::0.831470 0.555570
+DEAL::0.923880 0.382683
+DEAL::0.980785 0.195090
+DEAL::0.707107 0.707107
+DEAL::0.195090 0.980785
+DEAL::0.382683 0.923880
+DEAL::0.555570 0.831470
+DEAL::=================================
+DEAL::=================================
+DEAL::0.195090 0.980785
+DEAL::0.382683 0.923880
+DEAL::0.555570 0.831470
+DEAL::0.707107 0.707107
+DEAL::0.831470 0.555570
+DEAL::0.923880 0.382683
+DEAL::0.980785 0.195090
+DEAL::=================================
+DEAL::0.831470 0.555570
+DEAL::0.923880 0.382683
+DEAL::0.980785 0.195090
+DEAL::0.707107 0.707107
+DEAL::0.195090 0.980785
+DEAL::0.382683 0.923880
+DEAL::0.555570 0.831470
+DEAL::=================================
+DEAL::=================================
+DEAL::0.195090 0.00000 0.980785
+DEAL::0.382683 0.00000 0.923880
+DEAL::0.555570 0.00000 0.831470
+DEAL::0.707107 0.00000 0.707107
+DEAL::0.831470 0.00000 0.555570
+DEAL::0.923880 0.00000 0.382683
+DEAL::0.980785 0.00000 0.195090
+DEAL::=================================
+DEAL::0.831470 0.00000 0.555570
+DEAL::0.923880 0.00000 0.382683
+DEAL::0.980785 0.00000 0.195090
+DEAL::0.707107 0.00000 0.707107
+DEAL::0.195090 0.00000 0.980785
+DEAL::0.382683 0.00000 0.923880
+DEAL::0.555570 0.00000 0.831470
+DEAL::=================================
+DEAL::=================================
+DEAL::1.22465e-16 1.41421 1.41421
+DEAL::0.165159 1.40938 1.40938
+DEAL::0.329189 1.39493 1.39493
+DEAL::0.490971 1.37094 1.37094
+DEAL::0.649399 1.33759 1.33759
+DEAL::0.803391 1.29510 1.29510
+DEAL::0.951895 1.24376 1.24376
+DEAL::1.09390 1.18393 1.18393
+DEAL::1.22843 1.11601 1.11601
+DEAL::1.35456 1.04047 1.04047
+DEAL::1.47145 0.957821 0.957821
+DEAL::1.57828 0.868628 0.868628
+DEAL::1.67433 0.773502 0.773502
+DEAL::1.75895 0.673091 0.673091
+DEAL::1.83155 0.568083 0.568083
+DEAL::1.89163 0.459194 0.459194
+DEAL::1.93880 0.347169 0.347169
+DEAL::1.97272 0.232772 0.232772
+DEAL::1.99317 0.116785 0.116785
+DEAL::2.00000 0.00000 0.00000
+DEAL::=================================
+DEAL::=================================
+DEAL::0.612372 0.612372 0.500000
```

Comment:
@luca-heltai 
This is an issue. Given three points P1, P2, P3 and three weights w1,w2,w3 the order of the weighted sum is not commutative: w1P1 + w2P2 + w3P3  \not= w3P3 + w1P1 + w2P2